README.md

```diff
@@ -17,8 +17,8 @@
 ```yml
 # Configuration for linter - https://github.com/probot/linter
 
-# a whitList of files the linter will NOT be applied to
-whiteList:
+# a list of files the linter will NOT be applied to
+filesNotToLint:
```

Comment:
Good call, and thanks for pointing out the horribleness of this word. I never thought about it, but will try to never use it again.

What about `exclude`?